dotcom-rendering/src/web/components/Callout/Callout.tsx

```diff
@@ -43,8 +43,9 @@ const tabIcons = css`
 `;
 
 export interface CalloutBlockProps {
-	heading: string;
-	description: string;
+	prompt?: string;
+	heading?: string;
+	description?: string;
```

Comment:
Do these need to be optional? Can the non-optional string accept the empty string as a value?